pkgs/desktops/maxx/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv, fetchurl
+, libX11, libXext, libXi, libXau, libXrender, libXft, libXmu, libSM, libXcomposite, libXfixes, libXpm
+, libXinerama, libXdamage, libICE, libXtst
+, fontconfig, pango, cairo, glib, libxml2, atk, gtk2, gdk_pixbuf, mesa_noglu, ncurses5 }:
+
+let
+  version = "Indy-1.0.0";
+
+  deps = [
+    libX11 libXext libXi libXau libXrender libXft libXmu libSM libXcomposite libXfixes libXpm
+    libXinerama libXdamage libICE libXtst
+    stdenv.cc.cc fontconfig pango cairo glib libxml2 atk gtk2 gdk_pixbuf mesa_noglu ncurses5
+  ];
+in stdenv.mkDerivation {
+  name = "MaXX-${version}";
+
+  srcs = [
+    (fetchurl {
+      url = "http://maxxinteractive.com/downloads/${version}/FEDORA/MaXX-${version}-NO-ARCH.tar.gz";
```

Comment:
Please ask upstream to provide sources in a location not mentioning the name of a Linux distribution.